_dashboards/discover/visualizing-ad.md

```diff
@@ -0,0 +1,158 @@
+
+Alerts and anomalies are closely related, but they serve different purposes. Alerts inform you of issues that require immediate attention, while anomalies help you detect issues that may not yet have triggered an alert. By using both alerts and anomalies in your monitoring system, you can proactively manage and troubleshoot your system. 
+
+In this tutorial, you will learn to perform anomaly detection using the **Discover** application and line chart visualizations with OpenSearch sample data. At the end of this tutorial, you should have a good idea of how to use the Discover application and visualizations to monitor your own data. Common use cases for alerting and anomaly detection include network behavior, application performance, and web application security.
+
+The following video provides a quick overview of the steps performed in this tutorial:
+
+<insert demo from SME>
+
+
+## Getting started
+
+The following is useful terminology to understand before getting started with this tutorial:
+
+- _Anomaly detection_ is a technique used in data analysis to identify patterns or data points that deviate from the norm or expected behavior. It can be performed in real time, near real time, or on a scheduled basis.
+- _Alert_ refers to an event associated with a trigger. When an alert is created, the trigger performs actions, which can include sending a notification.. 
+- _Monitor_ refers to jobs that run on a defined schedule and query OpenSearch indexes.
+- _Trigger_ refers to conditions that generate alerts.
+- _Action_ refers to the information that you want the monitor to send out after being triggered. Actions have a destination, a message subject, and a message body.
+- _Destination_ refers to a reusable location for an action. Supported locations are Slack, email, or custom webhook.
+
+For an overview of the OpenSearch Alerting and Anomaly Detection plugins, see the OpenSearch documentation for [Alerting]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/index/) and [Anomaly Detection]({{site.url}}{{site.baseurl}}/observing-your-data/ad/index/).
+{: .note} 
+
+## Prerequisites 
+
+This tutorial has the following prerequisites: 
+
+- You must be running OpenSearch Dashboards before proceeding with the tutorial. 
+- You must be connected to your local OpenSearch Dashboards environment or `https://localhost:5601`. The username and password are `admin`.
+- You have basic familiarity with <prereq?>
+- <Other prereq?>
+
+## Configuring admin settings
+
+ Access to alerting and anomaly detection using visualizations is controlled by OpenSearch and OpenSearch Dashboards privileges. **Stack Management** is the place where you manage advanced settings. The setting is enabled by default and appears as a feature in the **Stack Management > Advanced Settings > Visualizations** window as shown below. If the setting is disabled, it does not appear in this window. The following image shows the enabled settings in the Advanced Settings window.
```

Comment:
> If the setting is disabled, it does not appear in this window.

I'm confused by this sentence. Currently there's no fine-grained access within advanced settings such that some settings are visible/read-only and others are not.

Reply:
Deleted